Automattic/woocommerce-payments
# Track WooPay button location updates

#### Changes proposed in this Pull Request





This PR adds a Tracks event that gets fired when updating WooPay express button location from the WCPay settings page. 
 




#### Testing instructions





- In WP Admin, go to **Payments > Settings**.
- From the "Express checkouts" section, go to WooPay -> Customize
- From the checkboxes under "Enable WooPay button on selected pages", enable WooPay for checkout and product pages and save the settings. 
- In ~5 minutes, checks the Tracks Live View for an event called `wcadmin_woopay_express_button_locations_updated`.
- Check the event properties for the presence of truthy values `checkout_enabled` and `product_enabled` props.
- Select a different combination of locations in "Enable WooPay button on selected pages" section and save settings
- In ~5 minutes, checks the Tracks Live View to ensure the that the props in  `wcadmin_woopay_express_button_locations_updated` matches the locations enabled in the settings page.

-------------------


**Post merge**
